x-pack/platform/plugins/shared/dataset_quality/public/components/dataset_quality_details/quality_issue_flyout/expandable_truncated_text.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import { EuiCode, EuiFlexGroup } from '@elastic/eui';
+import { readLess, readMore } from '../../../../common/translations';
+
+interface TruncatedTextWithToggleProps {
+  text: string;
+  maxCharLength?: number;
```

Comment:
Why do we have `maxCharLength` and `truncatedTextLength`?
I think we only should have one of the two, we should truncate the text if length is higher than `X`, and then use `X` to do the slicing.

Reply:
With @patpscal we decided that it would be better to truncate the message after certain amount of chars, in this case 150 and when it's truncated it also would be good to keep the message in one line (`truncatedTextLength`).

Reply:
Got it now, thanks!